### ISSUE TITLE
Show/Hide Sidebar popup labels reversed, button hides last item control

### ISSUE THREAD

**Issue by LorenAmelang** [zadam/trilium#1196]
App version: | 0.43.3

Maybe I'm missing something...  I needed to access the bottom item in my sidebar, but the << "Show Sidebar" button covers the down arrow that would expand it. I made a "Placeholder" note at the bottom to push my content above the button: 

![Sidebar Button](https://user-images.githubusercontent.com/1297090/90703186-f6d30e80-e241-11ea-9c58-db161a28a4f5.JPG)

Is there some better way? Maybe move it to the bottom right of the sidebar where it won't cover down arrows? 

Also, when I click << "Show Sidebar", the sidebar disappears. Click >> "Hide Sidebar" and it comes back. Text seems reversed...

**Comment by zadam** [comment]
Hello, thanks for reporting these issues:

1.  left sidebar toggle had reversed hide/show titles as you described, this has been fixed.
2.  I understand the issue with the toggle but moving it to the right is not simple - because the button needs to be visible also in the collapsed state it can't be positioned relatively to the sidebar but to the whole window. It's solvable, but a bit more complicated. Instead for now I created a workaround where it will be now possible to scroll a little bit below the last item, like this: 

![](https://user-images.githubusercontent.com/617641/90751700-bffb0800-e2d6-11ea-9d58-a0719f0f7cfc.png)
